test/cmake/target_js/CMakeLists.txt

```diff
@@ -4,21 +4,15 @@ project(test_cmake)
 
 add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/../cpp_lib ${CMAKE_CURRENT_BINARY_DIR}/cpp_lib)
 
-file(GLOB sourceFiles main.cpp)
-
-file(GLOB preJsFiles pre*.js)
-file(GLOB postJsFiles post*.js)
-file(GLOB libraryJsFiles jslibrary*.js)
-
 if (CMAKE_BUILD_TYPE STREQUAL Debug)
-  set(linkFlags "-g4 -sNO_EXIT_RUNTIME")
+  set(linkFlags "-g")
 else()
   # Either MinSizeRel, RelWithDebInfo or Release, all which run with optimizations enabled.
-  set(linkFlags "-O2 -sNO_EXIT_RUNTIME")
+  set(linkFlags "-O2")
 endif()
 
 # Ensure synchronous startup for this test, whose output expects it
-set(linkFlags "${linkFlags} -sWASM_ASYNC_COMPILATION=0")
+set(linkFlags "${linkFlags} -sNO_EXIT_RUNTIME -sWASM_ASYNC_COMPILATION=0")
```

Comment:
NO_EXIT_RUNTIME is the default now, right? so this can be left out?

Reply:
Done